### DISCUSSION TITLE
Toggle media content

### DISCUSSION THREAD

**Discussion by lubomsky** [ChurchApps/ChurchAppsSupport#313] [FreeShow] [answered]
Hello!
Awesome app - thank you!

It would be nice if single click on media (background images or videos) would toggle play, instead of preview. Like slides on show.
Maybe another option in settings which would change the behavior?
Option for loops by default would be handy too. Churches often have like 30 seconds of background and they are looping whole song.

**Comment by vassbo** [comment, marked as answer]
Hi, thank you. I think you are right. I can change the behaviour.
Also, videos applied to slides as a background are looped by default, but if you press play from the drawer you normally want it to not loop.
